config/locales/en.yml

```diff
@@ -35,7 +35,7 @@ en:
     open_apps: All open applications
 
   labels: &LABELS
-    assigned_to: Assigned to
+    assigned_to: 'Assigned to:'
```

Comment:
Do we need apostrophes here? I think remove them unless we do for some reason

Reply:
The colon makes the yml think it's another key, so need to be in quotes. Other option apparently to do:
key: >
  Value that has: colon
  
But the quotes seems cleaner